### PR TITLE
Topic/polygon styles

### PR DESCRIPTION
**Pull request by ahinz**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 *.idea
 *.pyc
 *.db
+.#*
```
Comment (steventlamb) [inline .gitignore:4]: is this for emacs recovery files? If so, consider using a personal gitignore.

File: templates/polygons/edit.html [hunk 2]
```diff
@@ -1,17 +1,11 @@
...
-    {% load compressed %}
-    {% compressed_js 'map' %}
-<link rel="stylesheet" href="{{ STATIC_URL }}css/polygons.css" type="text/css"/>
-{% endblock %}
-
-{% block middle_panel %}
+{% block polygoncontent %}
 
-<h1>Polygon #{{ polygon_id }}</h1>
+<h1>Polygon #{{ polygonobj.pk }}</h1>
```
Comment (steventlamb) [inline templates/polygons/edit.html:5]: In this case, is polygonobj a reference to a Polygon model instance? consider 'polygon'.

**Comment by steventlamb** [comment]
+1

File: templates/polygons/edit.html (hunk 2 above)
Comment (ahinz) [inline templates/polygons/edit.html:5]: that variable name was already in use

**Pull request merged**